index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>PORTFOLIO</title>
+</head>
+<body>
+    <header>
+        <ul class="toolbar">
+            <li class="welcome"><a href="#">Welcome</a></li>
+            <li class="menu"><a href="#">☰</a></li>
+
+        </ul>
+    </header>
+    <div class="headline">
+        <h1>I’m Amine Glade to see you!</h1>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <span><a>LET'S CONNECT <br> </a></span>
+        <img src="/social-media-icons.png" alt="Social Media icons">
+    </div>
```

Comment:
> * [ ]  Kindly use a `section` tag instead of the `div` tag here to make your page semantic. A semantic element clearly describes its meaning to both the browser and the developer. +1

- [ ] Please work on the above-suggested changes.